code_search/code-search.ipynb

```diff
@@ -0,0 +1,509 @@
+      "  \"private_key_id\": \"7fa99e3b3c8803b32beb6f2b0a147410bc806eec\",\n",
+      "  \"private_key\": \"-----BEGIN PRIVATE KEY-----\\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC1OUvIB9hWVCQu\\nJ/MdU7XH87sA4/iPT3cKjirIxTgT7kA5jXNnpPet0G4OhpjRje9Em+oIpdMcg8sX\\nv8OuluagUWi8UsU0naP0v8y8Uc2onYtaC59nCn+vg9EAyCutpT2wJeZH6mDH2Kta\\nubDh6NcobWqwCfqNS7tF7DYaxl/EQ3mvAt+TlAbPnSjRP7POIEE4DzOl3QuIZ9AN\\nbWiPn0Y/2+cpRkPqDIRAKDTBLAGcSIF//7aMSSalPh15dyGgdKPvF/SMyl+xiXQ9\\nA4p4k7UPzZcUpWiOvAwIkncs0Q1VnYuIuAvjj/u3ufFREInAlqz4ECZo0GQbbIaF\\n3U8P448ZAgMBAAECggEAQs8XHWyq+BR37B4lNcQVCVxUrfzdNvP8NkN4CWEPjeVw\\n/uajS2vZNVZYJHnBX8u8ECaMjliXrfT2S9CR0szlw+ePPZIkCoQtG/8Ter+LmmRO\\nKcmMH+ASd4GYbPnehFsdFVG7hfqlaDd74GwBhh8hJtHDmZdsK2fmZ94vigpk5sS8\\ndrdmSXRyrh8epvV54z/zZ6gIE3HrBbZD1nGJg81TOPtPQBwub+MCtm6Wg6sUsFdc\\nxDvC3ExIM3Nqyrrw71dbTNY/MVCZRQYSgQolcJraAFQuTPFmHrgtKeEXRV5/jAl+\\nhEDwf7o5U2NcnAD8Yq7VHMFyQP38T3fEKhEo7DlCGwKBgQDd8pmB2oDE/RFOmpqw\\nb+xJJQd/GRx+mtb3SVKm19sLAt+ANYDJMkChPSFbBBDkV3k4/K9Euq0Ahx22e9Y7\\nCyWz8IRLKGYd2e+HHtGVLAEtBRZXoTyCcyakoGncRx7YxOy6eWkn+cRZzO0Gxvfy\\nl1a2MGrusHoJE9E6/R/EqSETHwKBgQDRBzFkyrmwd+rMbBPv3oqHpB6BRuY8HiDu\\n7MnXnv7HPMS1SFeyGf0DXz0nlVgZmnyig0FzVrJz/TWeKrYRw3HYB6w9BIB1ALQ+\\n9XKHffpArVwJq6UXzQM2/nksitbqV1iCt5HuvYuERjQEyWOcJmRYy6JTLKkuB6He\\nTGYtpZ4OxwKBgEwy0gkG31bCG4MFCT6x7klecShtkp7UwiC8B6hIS2eIYPs/Yyuw\\nGQhCWIVK7BG5BmFOP041WNfpi4XvFinHNfaFCKZVBFoTTGzzY7j3FqBCMt16+a8/\\noXC7shxyPmDlmvCaZkPXOFYsSOQew1mCI78A+HViGUOsjhTWemWmOzi5AoGAQEB2\\nxWWvWHgz+2xJYJVGfdVL7y8M/HPCac7taFMeO74JNTQsiLmGVXAEC1PTxeL/5rB0\\nAPEoX5D410qHtsQeDWqmXgcxOUXjIesurkliEFl5jhtt3vgMwe3M740p7NS2h0/I\\nEtu+tAUkXkhNTyVZ+RV3KxvyUQzVi9BuJB/hiucCgYEAuSyOS3fhZASLbhi1DbsB\\nxzAQkGSwHA7TDdr7jxOD1LLetvN6nfjBj/KjsUg1C99K8R61EbZyH3+jLAkyzskj\\nu8M0ozPOyw18rxTWy6pgTPRJ+5XiD96Tf8BDoI1xTodGJGHzqzHEfOx0sMVh95QX\\nz0U0nta7qbBELSs6MgyTmjo=\\n-----END PRIVATE KEY-----\\n\",\n",
+      "  \"client_email\": \"kubeflow-latest-user@kubeflow-dev.iam.gserviceaccount.com\",\n",
+      "  \"client_id\": \"112957758415637004516\",\n",
+      "  \"auth_uri\": \"https://accounts.google.com/o/oauth2/auth\",\n",
+      "  \"token_uri\": \"https://accounts.google.com/o/oauth2/token\",\n",
+      "  \"auth_provider_x509_cert_url\": \"https://www.googleapis.com/oauth2/v1/certs\",\n",
+      "  \"client_x509_cert_url\": \"https://www.googleapis.com/robot/v1/metadata/x509/kubeflow-latest-user%40kubeflow-dev.iam.gserviceaccount.com\"\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "# FIXME: The deployment must set up the service account with GKE permissions\n",
+    "# gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}\n",
+    "\n",
+    "# gcloud container clusters get-credentials ${CLUSTER_NAME} --region ${CLUSTER_REGION}\n",
+    "# kubectl config view\n",
+    "# ks env add code-search --context=$(kubectl config current-context) --namespace=kubeflow\n",
+    "cat ${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
Delete this CL you are printing out the private key which gets stored in the notebook and then likely gets pushed to git  as is evident here.

This compromises the key.

Reply:
Oh that's terrible. I was in the middle of a cluster deployment and did not want to lose progress. Let me fix this commit.